scripts/binary/binary-integrity-check-source.js

```diff
@@ -0,0 +1,168 @@
+const OrigError = Error
+const captureStackTrace = Error.captureStackTrace
+const toString = Function.prototype.toString
+const callFn = Function.call
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
is this disable check needed? we are using it below without conditional checks

Reply:
Ah yeah. This can be removed.

Reply:
Removed: [`5c1f981` (#24909)](https://github.com/cypress-io/cypress/pull/24909/commits/5c1f98154e643a23ccddc01e3c02d7700059fcbe)